cmd/snap-update-ns/bootstrap.go

```diff
@@ -114,19 +114,21 @@ func validateInstanceName(instanceName string) int {
 //
 // This function is here only to make the C.validate_instance_name
 // code testable from go.
-func processArguments(args []string) (snapName string, shouldSetNs bool, processUserFstab bool) {
+func processArguments(args []string) (snapName string, shouldSetNs bool, processUserFstab bool, uid int) {
```

Comment:
.. and Golang which does not cover the full range of possible user IDs. I think we should use `uint` regardless of the type returned by functions in `os`.